sinonjs/samsam
# Exclude jsdom when bundling for headless tests

#### Purpose (TL;DR) - mandatory

Use Option `--no-detect-globals` for `mochify` command in `test-headless`

#### Background (Problem in detail)  - optional

Based on the suggestion by @mantoni [here](https://github.com/sinonjs/samsam/pull/130#issuecomment-541600189)

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm run test`, `npm run test-headless`

#### Checklist for author
